### PR TITLE
Make BitOperations public (CoreFX)

### PR DESCRIPTION
**Pull request by grant-d**
See related: https://github.com/dotnet/coreclr/pull/22864#issuecomment-467735960

This is the first time I'm working the CoreFX ref assemblies. 
Please let me know if/what I have missed.

cc @jkotas

### CODE REVIEW

**Comment by ahsonkhan** [comment]
> shouldCLSCompliantAttribute appear in ref assys (for new API in this case)

Yes.

**Comment by jkotas** [comment]
Could you please also move and fixup the tests ? I am disabling them in https://github.com/dotnet/corefx/pull/35615

**Comment by grant-d** [comment]
> Could you please also move and fixup the tests

Sure, will do

**Comment by grant-d** [comment]
@dotnet-bot test this please

**Comment by grant-d** [comment]
Build is failing due to:
`D:\a\1\s\.packages\microsoft.dotnet.genfacades\1.0.0-beta.19126.3\build\Microsoft.DotNet.GenFacades.targets(93,5): error : Did not find type 'T:System.Numerics.BitOperations' in any of the seed assemblies. [D:\a\1\s\src\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj]`

**Comment by tannergooding** [comment]
@grant-d, in addition to the changes mirroring back (#33615), we need a PR picking up a new build of CoreCLR (a newer PR of https://github.com/dotnet/corefx/pull/35586, that actually gets merged).

**Comment by grant-d** [comment]
Ah, thanks Tanner.

**Comment by grant-d** [comment]
@dotnet-bot test this please

File: src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj [hunk 1]
```diff
@@ -54,6 +54,7 @@
     <Compile Include="System\Runtime\Versioning\FrameworkName.cs" />
     <Compile Include="System\IO\PathTestsBase.cs" />
     <Compile Include="System\Net\WebUtility.cs" />
+    <Compile Include="System\Numerics\BitOperationsTests.cs" />
```
Comment (jkotas) [inline src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj]: Could you please move this to `'$(TargetGroup)'!='netstandard'` block to fix the netfx build flavor

**Comment by jkotas** [comment]
Also, could you please add `TypesMustExist : Type 'System.Numerics.BitOperations' does not exist in the implementation but it does exist in the contract.` to `src\System.Runtime.Extensions\src\ApiCompatBaseline.netcoreappaot.txt` and `src\System.Runtime.Extensions\src\ApiCompatBaseline.uapaot.txt` to fix the UWP_NETNative_x86_Release leg

File: src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj (hunk 1 above)
Comment (grant-d) [reply src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj]: Thanks - fixed

**Comment by jkotas** [comment]
Could you please also change GenFacadesIgnoreMissingTypes line in `src\System.Runtime.Extensions\src\ System.Runtime.Extensions.csproj` to:

```
<GenFacadesIgnoreMissingTypes Condition="'$(TargetsAOT)'=='true' OR '$(TargetGroup)' == 'uap'">true</GenFacadesIgnoreMissingTypes>
```

This should fix the "Packaging All Configuration".

The Linux failure looks like an intermittent bug. Filled https://github.com/dotnet/corefx/issues/35708

**Comment by grant-d** [comment]
Thanks again for your help Jan

**Pull request merged**